core/src/avm2/globals/flash/display/NativeMenu.as

```diff
@@ -1,6 +1,144 @@
-package flash.display {
+package flash.display
+{
+
     import flash.events.EventDispatcher;
-    public class NativeMenu extends EventDispatcher {
+    import __ruffle__.stub_method;
+    import __ruffle__.stub_getter;
+    
+    // According to the documentation, it should be [API("661")]
+    // but airglobal.swf disagrees with that:
```

Comment:
nit: It's `airglobal.swc`, not `swf`. :sweat_smile:

Reply:
oops